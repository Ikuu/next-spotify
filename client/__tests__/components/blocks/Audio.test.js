import React from 'react';
import { create } from 'react-test-renderer';
import Audio from '../../../components/blocks/Audio';

test('Audio renders', () => {
  const tree = create(<Audio src="www.mock.com" />).toJSON();
  expect(tree).toMatchSnapshot();
});
