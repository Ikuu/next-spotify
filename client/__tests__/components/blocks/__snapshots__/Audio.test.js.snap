// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Audio renders 1`] = `
<audio
  controls={true}
>
  <source
    src="www.mock.com"
    type="audio/mp3"
  />
  Your browser does not support the Audio element.
</audio>
`;
